appveyor.yml

```diff
@@ -4,56 +4,44 @@ environment:
   global:
     # This variable used only if '--upload' added to 'jenkins.py'
     GITHUB_USER_PASSWORD:
-      secure: KbhVDnegXkjLm8H7lUZX9/t/4AL3dzhQLy8mWZ3Ldv0sXDs7d5bFz7+Yo3EFZNtw
+      secure: tzWauRyJgacWahxWX5cKP+rnnczqdEXAIzP7ayUgt16GfoGKpkqa/QvDJYxLKM6Q
 
   matrix:
-    # Note: 'default' toolchain is the same as 'vs-14-2015'
 
-     - TOOLCHAIN: "ninja-vs-12-2013-win64"
-       PROJECT_DIR: examples\MySQL-client
+    - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
+      PROJECT_DIR: examples\MySQL-client
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-     - TOOLCHAIN: "nmake-vs-12-2013-win64"
-       PROJECT_DIR: examples\MySQL-client
+    - TOOLCHAIN: "nmake-vs-15-2017-win64-cxx17"
+      PROJECT_DIR: examples\MySQL-client
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-     - TOOLCHAIN: "nmake-vs-12-2013"
-       PROJECT_DIR: examples\MySQL-client
+#    https://ci.appveyor.com/project/caseymcc/hunter/builds/19950045/job/kf5bbug53mwuv52y
+#    - TOOLCHAIN: "vs-15-2017-win64-cxx17"
+#      PROJECT_DIR: examples\MySQL-client
+#      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-     - TOOLCHAIN: "vs-10-2010"
-       PROJECT_DIR: examples\MySQL-client
+    - TOOLCHAIN: "vs-14-2015-win64"
```

Comment:
It's `vs-14-2015-sdk-8-1`, not `vs-14-2015-win64`:
* https://github.com/ingenue/hunter/blob/3650bcbc792940034cdbad7f7f05051dd5f1f68c/appveyor.yml#L23-L25

If you want to change toolchain then it's better to comment out default one and add another to the bottom.